server/src/WebApp/Controller/Registration.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace AllyTalks\WebApp\Controller;
+
+use AllyTalks\WebApp\Application;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+
+class Registration extends Controller
+{
+    private $app;
+    private $model;
+    private $render;
+
+    public function __construct(Application $application)
+    {
+        $this->app = $application;
+        $this->model = $this->app->getModel();
+        $this->render = new Render($application);
+    }
+
+    /**
+     * @controller
+     * @method POST
+     * @route /post/reg
+     *
+     * @param Request $request
+     * @return Response
+     */
+    public function registerNewUser(Request $request)
+    {
+        $nickname = $this->prepareData($request->request->get('nickname'));
+        $login = $this->prepareData($request->request->get('login'));
+        $password = $this->prepareData($request->request->get('password'));
+        $email = $this->prepareData($request->request->get('email'));
+
+        $errors = $this->assertUser($nickname, $login, $password, $email);
+        $data = array('nickname' => $nickname, 'login' => $login, 'email' => $email);
+        
+        if(!$errors) {
+            if ($this->model->addUser(array('login' => $login, 'password' => $password, 'nickname' => $nickname, 'email' => $email)))
+                return new Response('Thank you for your registration!');
+            else
```

Comment:
short array notation + format
please setup code autoformatter in your phpstorm [click](https://www.jetbrains.com/help/phpstorm/2016.1/reformatting-source-code.html?origin=old_help)
we're using symfony2 code style